src/todo/components/add-input/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @flow
+ */
+// $FlowFixMe  (Flow doesn't like importing html files)
```

Comment:
this might not be needed now that i've set up the `.html` file workaround in .flowconfig